### PR TITLE
[Feature] Event Section

### PR DESCRIPTION
**Pull request by falbas**

### CODE REVIEW

**Comment by viantonugroho11** [comment]
image
lah ko begini

File: src/pages/index.tsx [hunk 1]
```diff
@@ -3,8 +3,9 @@ import * as React from 'react'
 import { Layout } from 'src/layouts/Main/Layout'
 
 import { Seo } from '@/elements/Seo/Seo'
-import Event from '@/modules/Event'
-import Intro from '@/modules/Intro'
+import { Intro } from '@/modules/Intro/Intro'
```
Comment (viantonugroho11) [inline src/pages/index.tsx:8]: image
Eror Disini

**Review by viantonugroho11 — CHANGES_REQUESTED** [review]

**Comment by falbas** [comment]
@viantonugroho11 pull ulang
di gue aman soalnya

**Comment by falbas** [comment]
image

**Comment by viantonugroho11** [comment]
Oke aman

File: src/types/type.ts [hunk 2]
```diff
@@ -33,3 +33,21 @@ export type Division = {
     email: string
   }[]
 }
+
+export type EventListAPI = {
```
Comment (gregoriusjimmy) [inline src/types/type.ts]: if this type represent one single event object, please rename it to `Event`

File: src/components/modules/OurDivision/OurDivision.tsx [hunk 3]
```diff
@@ -11,6 +11,8 @@ type Props = {
 }
 
 export const OurDivision = ({ divisions, className }: Props) => {
+  // console.log(divisions)
```
Comment (gregoriusjimmy) [inline src/components/modules/OurDivision/OurDivision.tsx]: remove this line

File: src/components/modules/EventList/EventList.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
```
Comment (gregoriusjimmy) [inline src/components/modules/EventList/EventList.tsx]: remove unused import

File: src/pages/index.tsx [hunk 5]
```diff
@@ -44,7 +49,19 @@ export async function getStaticProps() {
 
   const divisions = await getDivision()
 
+  const getEventList = async (): Promise<EventListAPI[]> => {
```
Comment (gregoriusjimmy) [inline src/pages/index.tsx]: rename this to `getEvents`

File: src/pages/index.tsx [hunk 6]
```diff
@@ -44,7 +49,19 @@ export async function getStaticProps() {
...
+    try {
+      const resEventList = await axios.get<GetEventListResponse>('event')
```
Comment (gregoriusjimmy) [inline src/pages/index.tsx]: rename res or resEvents

File: src/pages/index.tsx [hunk 7]
```diff
@@ -3,27 +3,32 @@ import * as React from 'react'
...
 import { OurDivision } from '@/modules/OurDivision/OurDivision'
 
+import { GetEventListResponse } from '@/types/response'
+import { EventListAPI } from '@/types/type'
 import { GetDivisionResponse } from '@/types/response'
 import { Division } from '@/types/type'
 
 type Props = {
   divisions: Division[]
+  eventLists: EventListAPI[]
```
Comment (gregoriusjimmy) [inline src/pages/index.tsx]: events

File: src/components/modules/EventList/EventList.tsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+import { NextImage } from '@/elements/NextImage/NextImage'
+import Link from 'next/link'
+
+import { EventListAPI } from '@/types/type'
+
+type Props = {
+  eventLists: EventListAPI[]
+}
+
+export const EventList = ({ eventLists }: Props) => {
```
Comment (gregoriusjimmy) [inline src/components/modules/EventList/EventList.tsx]: so we will change this to ArticleSection

File: src/components/modules/Event/Event.tsx [hunk 9]
```diff
@@ -1,7 +1,7 @@
 import { NextImage } from '@/elements/NextImage/NextImage'
 import Link from 'next/link'
 
-export default function Event() {
+export const Event = () => {
```
Comment (gregoriusjimmy) [inline src/components/modules/Event/Event.tsx]: rename this to EventSection to reduce confusion between the Event data

**Review by gregoriusjimmy — APPROVED** [review]
CAKEPP MERGE!!

**Comment by viantonugroho11** [comment]
Mantap HAN

**Pull request merged**